### PR TITLE
mgr/orchestrator: Update docstring of service_action method

### PR DESCRIPTION
**Pull request by varshar16**
Signed-off-by: Varsha Rao <varao@redhat.com>

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/orchestrator/_interface.py [hunk 1]
```diff
@@ -956,8 +956,8 @@ def service_action(self, action, service_name):
         providing the logical service).
 
         :param action: one of "start", "stop", "restart", "redeploy", "reconfig"
-        :param service_type: e.g. "mds", "rgw", ...
-        :param service_name: name of logical service ("cephfs", "us-east", ...)
+        :param service_name: service_type (e.g. "mds", "rgw", ...)  + name of
+        logical service ("cephfs", "us-east", ...)
```
Comment (sebastian-philipp) [inline src/pybind/mgr/orchestrator/_interface.py:960]:
```suggestion
        :param service_name: (e.g. "mon", "mgr", "mds.mycephfs", "rgw.realm.zone", ...)
```
?

Comment (varshar16) [reply src/pybind/mgr/orchestrator/_interface.py]: I have modified it a bit.

**Comment by Devp00l** [comment]
Hi I just wondered about your commit description "...n methods"  and I though the commit title is so long that github won't show it because of "..." at the end ;) After clicking on the commit present with a view where title and message aren't shown as far apart it was clear to me that it should be displayed together.

That said how about to shorten the message?
Instead of:

```
mgr/orchestrator: Update docstring of service_action and daemon_actio…

…n methods
```

How about:
```
mgr/orchestrator: Updates service_action and daemon_action docstrings
```

**Comment by varshar16** [comment]
> Hi I just wondered about your commit description "...n methods" and I though the commit title is so long that github won't show it because of "..." at the end ;) After clicking on the commit present with a view where title and message aren't shown as far apart it was clear to me that it should be displayed together.
> 
> That said how about to shorten the message?
> Instead of:
> 
> ```
> mgr/orchestrator: Update docstring of service_action and daemon_actio…
> 
> …n methods
> ```
> 
> How about:
> 
> ```
> mgr/orchestrator: Updates service_action and daemon_action docstrings
> ```

I have updated the commit title.

**Comment by varshar16** [comment]
jenkins retest this please

**Comment by sebastian-philipp** [comment]
green: https://pulpito.ceph.com/swagner-2020-08-04_10:19:43-rados:cephadm-wip-swagner-testing-2020-08-04-1029-distro-basic-smithi/

File: src/pybind/mgr/orchestrator/_interface.py [hunk 2]
```diff
@@ -956,8 +956,8 @@ def service_action(self, action, service_name):
         providing the logical service).
 
         :param action: one of "start", "stop", "restart", "redeploy", "reconfig"
-        :param service_type: e.g. "mds", "rgw", ...
-        :param service_name: name of logical service ("cephfs", "us-east", ...)
+        :param service_name: service_type + '.' + service_id
+        (e.g. "mon", "mgr", "mds.mycephfs", "rgw.realm.zone", ...)
         :rtype: Completion
```
Comment (sebastian-philipp) [inline src/pybind/mgr/orchestrator/_interface.py:961]: /home/jenkins-build/build/workspace/ceph-pr-docs/src/pybind/mgr/orchestrator/_interface.py:docstring of orchestrator.Orchestrator.service_action:8: WARNING: Field list ends without a blank line; unexpected unindent.

https://jenkins.ceph.com/job/ceph-pr-docs/29817/consoleFull#-85362248744e9240e-b50a-4693-bac0-8a991bac86ac

File: src/pybind/mgr/orchestrator/_interface.py [hunk 3]
```diff
@@ -969,7 +969,9 @@ def daemon_action(self, action, daemon_type, daemon_id):
         Perform an action (start/stop/reload) on a daemon.
 
         :param action: one of "start", "stop", "restart", "redeploy", "reconfig"
-        :param name: name of daemon
+        :param daemon_type: type of daemon
+        :param daemon_id: Depending on daemon type it can be numerical value or
+        service name
         :rtype: Completion
```
Comment (sebastian-philipp) [inline src/pybind/mgr/orchestrator/_interface.py:975]: /home/jenkins-build/build/workspace/ceph-pr-docs/src/pybind/mgr/orchestrator/_interface.py:docstring of orchestrator.Orchestrator.daemon_action:9: WARNING: Field list ends without a blank line; unexpected unindent.

**Comment by sebastian-philipp** [comment]
needs rebase

**Pull request merged**